packages/langium/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A language engineering tool for the Language Server Protocol",
   "homepage": "https://langium.org",
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.18.0"
```

Comment:
We may need to name an npm version here as well, because the `package-lock.json` version is already `"lockfileVersion": 3` which requires I guess npm v9 (need to find that out)

Reply:
That's declared in the root `package.json`, see https://github.com/eclipse-langium/langium/blob/79e430127324c5ff45a2c753744ac1aa7c5237e0/package.json#L6

Reply:
Overlooked that one.